Source/Kassiopeia/LMCFieldCalculator.hh

```diff
@@ -32,17 +32,18 @@ namespace locust
             bool ConfigureByInterface();
 
 
+            bool ModeSelect(int l, int m, int n, bool eGun, bool bNormCheck);
             double GetGroupVelocityTM01(Kassiopeia::KSParticle& aFinalParticle);
             double GetGroupVelocityTE10(Kassiopeia::KSParticle& aFinalParticle);
             double GetDampingFactorPhase2(Kassiopeia::KSParticle& aFinalParticle);
             double GetDampingFactorPhase1(Kassiopeia::KSParticle& aFinalParticle);
             double GetDampingFactorCavity(Kassiopeia::KSParticle& aFinalParticle);
             double GetCouplingFactorTM01(Kassiopeia::KSParticle& aFinalParticle);
             double GetCouplingFactorTE10(Kassiopeia::KSParticle& aFinalParticle);
-            double GetCouplingFactorTE011Cavity(Kassiopeia::KSParticle& aFinalParticle);
+            double GetCouplingFactorTElmnCavity(int l, int m, int n, bool bTE, Kassiopeia::KSParticle& aFinalParticle);
             double GetTM01FieldWithTerminator(Kassiopeia::KSParticle& aFinalParticle);
             double GetTE10FieldAfterOneBounce(Kassiopeia::KSParticle& aFinalParticle);
-            double GetTE011FieldCavity(Kassiopeia::KSParticle& aFinalParticle);
+            double GetTElmnFieldCavity(int l, int m, int n, Kassiopeia::KSParticle& aFinalParticle);
```

Comment:
Similarly we'll want to generalize this to both TE and TM modes soon, so it's probably worth removing the 'TE' from this function name and add a 'bool bTE' input here too, even if it isn't currently utilized in the actual function definition